docs/release_notes/v1.0.0-rc.2.md

```diff
@@ -0,0 +1,538 @@
+
+Once Dapr is installed completely, ensure until **all pods are running**
+```sh
+kubectl get pods -n dapr-system -w
+
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-69f5c5c867-mqhg4          1/1     Running   0          42s
+dapr-operator-5cdd6b7f9c-9sl7g           1/1     Running   0          41s
+dapr-operator-5cdd6b7f9c-jkzjs           1/1     Running   0          29s
+dapr-operator-5cdd6b7f9c-qzp8n           1/1     Running   0          34s
+dapr-placement-server-0                  1/1     Running   0          41s
+dapr-placement-server-1                  1/1     Running   0          41s
+dapr-placement-server-2                  1/1     Running   0          41s
+dapr-sentry-84565c747b-7bh8h             1/1     Running   0          35s
+dapr-sentry-84565c747b-fdlls             1/1     Running   0          41s
+dapr-sentry-84565c747b-ldnsf             1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-6xnbt   1/1     Running   0          41s
+dapr-sidecar-injector-68f868668f-j7jcq   1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-ltxq4   1/1     Running   0          36s
+```
+
+
+Restart your application deployments to update the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+All done!
+
+#### Starting fresh install on a cluster 
+
+You can use Helm 3 to install Dapr:
+```sh
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+kubectl create namespace dapr-system
+helm install dapr dapr/dapr --version 1.0.0-rc.2 --namespace dapr-system
+```
+
+Alternatively, you can use the latest version of CLI:
+
+```
+dapr init --runtime-version=1.0.0-rc.2 --kubernetes
+```
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI version*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+##### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ kubectl get pods -n dapr-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+dapr-dashboard-85cc6c9f7d-hzk69         1/1     Running   0          13m
+dapr-operator-84985cb985-h2zdv          1/1     Running   0          13m
+dapr-placement-server-0                 1/1     Running   0          13m
+dapr-sentry-5974c7cb6d-9bnqd            1/1     Running   0          13m
+dapr-sidecar-injector-5b5bb9454-sfmmx   1/1     Running   0          13m
+```
+
+After Dapr 1.0.0-rc.2 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
```

Comment:
This post installation is nested within starting a new cluster. Does the rollout command make sense here?

Reply:
Makes sense. Removed the rolling restart instruction.